pinc/graph_data.inc

```diff
@@ -13,6 +13,14 @@ function get_graph_js_files()
     ];
 }
 
+function build_svg_graph_inits($graphs)
+{
+    foreach ($graphs as [$type, $id, $config]) {
+        $js_data[] = "$type('$id', " . json_encode($config) . ");";
+    }
+    return "$(function(){" . "".join($js_data) . "});";
```

Comment:
I don't actually think `"".join()` works -- I was channeling python I guess. You want `join($js_data)`.

Oh! The reason it works is because it's just concatenating an empty string. Yeah, you can remove `"".` here.